less/modals.less

```diff
@@ -0,0 +1,37 @@
+//
+// Modals
+// --------------------------------------------------
+
+
+// Actual modal
+.modal-content {
+  border-top: 4px solid @brand-primary; //old browsers fallback (ie8 etc)
+  border-top: 4px solid @brand-primary;
+    .box-shadow(0 3px 9px rgba(0,0,0,.5));
+}
+
+
+// Footer (for actions)
+.modal-footer {
+  background: @modal-footer-bg;
+  // border-top: 1px solid @modal-footer-border-color;
+  &:extend(.clearfix all); // clear it in case folks use .pull-* classes on buttons
+
+  // Properly space out buttons
+  .btn + .btn {
+    margin-left: 5px;
+    margin-bottom: 5px; // account for input[type="submit"] which gets the bottom margin like all other inputs
```

Comment:
We should remove the comment on this line since it was commenting on the fact that they were setting the `margin-bottom` to `0`, which you're reverting.

(I don't completely understand the original comment (see https://github.com/twbs/bootstrap/commit/d8041f7c8f5f6fcb7777383429464f39879ba916) seeing as we don't have any `input[type="submit"]` buttons in the modal and it looks odd without your change, but perhaps we'll find out later and have to add this back. :))